test/verify/check-shell-multi-machine

```diff
@@ -451,7 +451,7 @@ class TestMultiMachine(MachineCase):
         m2 = self.machine2
 
         # Modify the terminals to be different on the two machines.
-        for machine, name in zip((m1, m2), ('m1', 'm2')):
+        for machine, name in zip((m1, m2), ('m1', 'm2'), strict=True):
```

Comment:
This requires a relatively new Python version....

Reply:
This should be okay. This is a destructive test, i.e. it only runs in our Fedora 38 tasks container, not in downstream gating in e.g. CentOS 8.

Reply:
Thanks for checking @martinpitt !